civis/resources/_resources.py

```diff
@@ -29,7 +29,9 @@
 with open(API_SPEC_PATH) as f:
     API_SPEC = json.load(f, object_pairs_hook=OrderedDict)
 BASE_RESOURCES_V1 = sorted(
-    set(path.split("/", 2)[1] for path in API_SPEC["paths"].keys())
+    r for r in set(path.split("/", 2)[1] for path in API_SPEC["paths"].keys())
+    # "feature_flags" has a name collision with an APIClient instance
+    if r != "feature_flags"
```

Comment:
I should've done this in #440. The codebase at v1.15.1 didn't have "feature_flags" either:

https://github.com/civisanalytics/civis-python/blob/v1.15.1/civis/resources/_resources.py#L24-L62

Also, "feature_flags" is explicitly filtered out when the class instances for the endpoints are generated (we're still doing this filtering):

https://github.com/civisanalytics/civis-python/blob/v1.15.1/civis/resources/_resources.py#L611-L612